README

```diff
@@ -2789,15 +2894,14 @@ The bibliography may have any of these formats:
   MODS              .mods
   RIS               .ris
 
-Note that `.bib` can generally be used with both BibTeX and BibLaTeX
-files, but you can use `.bibtex` to force BibTeX.
+Note that `.bib` can be used with both BibTeX and BibLaTeX files;
+use `.bibtex` to force BibTeX.
 
 Note that `pandoc-citeproc --bib2json` and `pandoc-citeproc --bib2yaml`
 can produce `.json` and `.yaml` files from any of the supported formats.
 
-In-field markup: In bibtex and biblatex databases, pandoc-citeproc parses
-(a subset of) LaTeX markup; in CSL JSON databases, an HTML-like markup
-([specs](http://docs.citationstyles.org/en/1.0/release-notes.html#rich-text-markup-within-fields));
+In-field markup: In BibTeX and BibLaTeX databases, pandoc-citeproc parses
+a subset of LaTeX markup; in CSL JSON databases, an [HTML-like markup][CSL markup specs];
```

Comment:
+++ Andrew Dunning [Oct 08 15 09:46 ]:
>   In [1]README:
>>
>>  Note that `pandoc-citeproc --bib2json` and `pandoc-citeproc --bib2yaml`
>>  can produce `.json` and `.yaml` files from any of the supported formats.
>>
>> -In-field markup: In bibtex and biblatex databases, pandoc-citeproc parses
>> -(a subset of) LaTeX markup; in CSL JSON databases, an HTML-like markup
>> -([specs](http://docs.citationstyles.org/en/1.0/release-notes.html#rich-text-m
>arkup-within-fields));
>> +In-field markup: In BibTeX and BibLaTeX databases, pandoc-citeproc parses
>> +a subset of LaTeX markup; in CSL JSON databases, an [HTML-like markup][CSL ma
>rkup specs];
>
>   The CSL specs do not include things such as <sc> or <span
>   class="nocase">, but I cannot seem to find an accessible listing of all
>   the available tags. Should one be added here directly?

We should probably document these two, anyway.